.github/workflows/python-tests.yml

```
@@ -30,7 +30,7 @@ jobs:
           - os: windows-latest
             python-version: '3.11'
             toxenv: py311-test-pytest72
-          - os: macos-13
+          - os: macos-14
```

3.11 should still be supported in latest?
```suggestion
          - os: macos-latest
```